lib/fake_api_response_wrapper.rb

```diff
@@ -0,0 +1,28 @@
+module FakeApiResponseWrapper
```

Comment:
this is now a module that could be mixed in with different fake system Api wrappers, since we anticipate this being consistent.